ilastik/shell/gui/licenseDialog.py

```diff
@@ -20,98 +20,119 @@
 
-        global ilastik_path
-        logo_path = os.path.join(ilastik_path, "shell", "gui", "ilastik-logo-alternate-colors.png")
+        # environment variable set in release binary
+        license_path = Path(os.environ.get("ILASTIK_LICENSE_PATH", ilastik_path / "LICENSE"))
+        license_3rd_party_path = Path(os.environ.get("ILASTIK_LICENSE_3RD_PARTY_PATH", "NOTFOUND"))
 
-        logo_image = QPixmap(logo_path).scaled(125, 200, Qt.KeepAspectRatio)
-        self.setIconPixmap(logo_image)
+        logo_path = ilastik_path / "ilastik" / "ilastik-fist.png"
 
-        license_path = os.path.join(ilastik_path, "..", "LICENSE")
-        # if ilastik is run from a binary
-        # use the COPYING file in the distribution
-        license_path_dist = get_buildem_license()
-        if license_path_dist:
-            license_long = open(license_path_dist, "r").read()
-        elif os.path.isfile(license_path):
-            license_long = open(license_path, "r").read()
-        else:
-            license_long = self.LICENSE_ERROR
+        logo_image = QPixmap(str(logo_path)).scaled(125, 200, Qt.KeepAspectRatio, Qt.SmoothTransformation)
+        logo = QLabel()
+        logo.setPixmap(logo_image)
 
-        self.setDetailedText(license_long)
+        short_license_label = QLabel(self.LICENSE_SHORT)
+        short_license_label.setWordWrap(True)
+        short_license_label.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding)
+        short_license_label.setMinimumSize(600, 50)
 
-        self.setWindowModality(Qt.NonModal)
-        self.open()
+        header_layout = QHBoxLayout()
+        header_layout.addWidget(logo, 0)
+        header_layout.addWidget(short_license_label, 1)
 
-    def closeEvent(self, event):
-        """
-        Emulate accept signal to close the window.
+        def show_license(_checked, license_path: Path, error_message: str):
+            if license_path.is_file():
+                webbrowser.open(license_path.as_uri())
+            else:
+                # parent, title, text
+                QMessageBox.warning(self, "License file not found!", error_message)
 
-        Using the window close button does not close a QMessageBox.
-        """
-        self.accept()
-        super(LicenseDialog, self).closeEvent(event)
+        show_details_btn = QPushButton("Show full license")
+        show_details_btn.clicked.connect(
+            partial(show_license, license_path=license_path, error_message=self.LICENSE_ERROR)
+        )
+        self._show_details_btn = show_details_btn
+        show_3rd_party_btn = QPushButton("Show third party licenses")
+        show_3rd_party_btn.clicked.connect(
+            partial(show_license, license_path=license_3rd_party_path, error_message=self.LICENSE_3RD_PARTY_ERROR)
+        )
+        self._show_3rd_party_btn = show_3rd_party_btn
+        btnbox = QHBoxLayout()
+        btnbox.addItem(QSpacerItem(0, 0, QSizePolicy.Expanding, QSizePolicy.Minimum))
```

Comment:
```suggestion
        btnbox.addStretch()
```

Reply:
looks like I've been using the designer too much...